drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -677,6 +679,7 @@ static void bcm2835_isp_node_stop_streaming(struct vb2_queue *q)
 				 "%s: Failed disabling component, ret %d\n",
 				 __func__, ret);
 		}
+		dev->last_ls_dmabuf = NULL;
```

Comment:
You could also update `dev->ls` instead of having a local here.

Is there a path in stopping and restarting where it reuses the existing `dev->ls` struct or control value, and is then using a stale memhandle? In theory it should be possible to disable a LS table after having been configured, but I'm not sure that is supported.

Reply:
I'll reuse ``dev->ls`` and null it.  That's probably the safest thing.  The libcamera client will always re-populate the fd on startup.